proto/internal/temporal/server/api/replication/v1/message.proto

```diff
@@ -165,6 +165,12 @@ message SyncActivityTaskAttributes {
     string last_started_build_id = 16;
     // workflows redirect_counter value when this activity started last time
     int64 last_started_redirect_counter = 17;
+    // First time the activity was scheduled.
+    google.protobuf.Timestamp first_scheduled_time = 18;
+    // Last time an activity failure was recorded by the server.
+    google.protobuf.Timestamp last_attempt_complete_time = 19;
+    // stamp represent "version" of activity during its lifetime. It can be changed with Activity API
```

Comment:
I had a suggestion above for this doc string. I still think the name `stamp` isn't super informative. The fact that you have to explain here that stamp represents a "version" shows the name can be improved.

Reply:
talked offline. will stick to `stamp` for now.